karapace/kafka_rest_apis/__main__.py

```diff
@@ -19,9 +26,14 @@ def main() -> int:
 
     logging.basicConfig(level=logging.INFO, format=DEFAULT_LOG_FORMAT_JOURNAL)
     logging.getLogger().setLevel(config["log_level"])
+    if config.get("access_logs_debug"):
```

Comment:
This branch will be taken if the value of the config is truthy, including the string `"false"`, should have validation for the type been `bool` to prevent unexpected behavior